ports/raspberrypi/boards/lilygo_t_display_rp2040/board.c

```diff
@@ -116,13 +116,13 @@ static void display_init(void) {
         &pin_GPIO4,     // backlight pin
         NO_BRIGHTNESS_COMMAND,
         1.0f,           // brightness (ignored)
-        false,          // auto_brightness
```

Comment:
I'm surprised the display worked at all with the argument positions shifted like this!